web/modules/custom/configform/configform.routing.yml

```diff
@@ -0,0 +1,15 @@
+Form.simpleForm:
+  path: '/form'
+  defaults:
+    _form: '\Drupal\configform\Form\SimpleForm'
+    _title: 'Simple Form'
+  requirements:
+    _permission: 'simple custom form permission'
+
+Form.configForm:
```

Comment:
config_form.config_form

Reply:
Upon clicking on configure this the form should open , so the key here and https://github.com/sandipta18/Drupal-Backend/pull/5#discussion_r1271708149 should be same right?